graphql/resolvers.go

```diff
@@ -2465,15 +2465,24 @@ func (r *taskResolver) ExecutionTasksFull(ctx context.Context, obj *restModel.AP
 	executionTasks := []*restModel.APITask{}
 	for _, execTaskID := range t.ExecutionTasks {
 		execT, err := task.FindByIdExecution(execTaskID, &t.Execution)
-		if err != nil || execT == nil {
+		if err != nil {
 			return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error while getting execution task with id: %s : %s", execTaskID, err.Error()))
 		}
-		apiTask := &restModel.APITask{}
-		err = apiTask.BuildFromService(execT)
-		if err != nil {
-			return nil, InternalServerError.Send(ctx, fmt.Sprintf("Unable to convert task: %s to APITask", execT.Id))
+		if execT != nil {
+			apiTask := &restModel.APITask{}
+			if err = apiTask.BuildFromService(execT); err != nil {
+				return nil, InternalServerError.Send(ctx, fmt.Sprintf("Unable to convert task: %s to APITask", execT.Id))
+			}
+			executionTasks = append(executionTasks, apiTask)
+		} else {
+			grip.Warning(message.Fields{
```

Comment:
Will not erroring here cause weird behavior for users? (like execution tasks are missing but it won't be obvious to them why)

Reply:
This is the kind of bug that would get lost in splunk for us unless someone asked us what had happened.

Reply:
Right, these code changes will allow the task table to render without the offending execution tasks and will log the task id's that were not found in the DB. I'm not really sure why the execution tasks weren't found but I hope a list of missing task id's can help us identify a similarity amongst the set to help us debug an issue in the task lifecycle or somewhere else.